packages/quantic/force-app/main/default/lwc/quanticSort/e2e/quanticSort.e2e.ts

```diff
@@ -0,0 +1,74 @@
+import {testSearch, testInsight, expect} from './fixture';
+import {useCaseTestCases} from '../../../../../../playwright/utils/useCase';
+
+const newestDateSort = 'date descending';
+const viewsDescendingSort = '@ytviewcount descending';
+const defaultSortLabels = ['Relevancy', 'Newest', 'Oldest'];
+
+const fixtures = {
+  search: testSearch,
+  insight: testInsight,
+};
+
+useCaseTestCases.forEach((useCase) => {
+  let test = fixtures[useCase.value];
+
+  test.describe(`quantic sort  ${useCase.label}`, () => {
+    test.describe(`when changing sort option to Newest`, () => {
+      test('should trigger a new search and log analytics', async ({
+        sort,
+        search,
+      }) => {
+        const searchResponsePromise = search.waitForSearchResponse();
+        await sort.clickSortDropDown();
+        await sort.clickSortButton('Newest');
+        const searchResponse = await searchResponsePromise;
+        const {sortCriteria} = searchResponse.request().postDataJSON();
+        expect(sortCriteria).toBe(newestDateSort);
+        await sort.waitForSortUaAnalytics(newestDateSort);
+      });
+    });
+
+    test.describe(`when changing custom sort to "Views Descending`, () => {
+      test('should trigger a new search and log analytics', async ({
+        sortCustom,
+        search,
+      }) => {
+        const searchResponsePromise = search.waitForSearchResponse();
+        await sortCustom.clickSortDropDown();
+        await sortCustom.clickSortButton('Views Descending');
+        const searchResponse = await searchResponsePromise;
+        const {sortCriteria} = searchResponse.request().postDataJSON();
+        expect(sortCriteria).toBe(viewsDescendingSort);
+        await sortCustom.waitForSortUaAnalytics(viewsDescendingSort);
+      });
+    });
+
```

Comment:
We should add tests for when the URL hash is changed to another sortCriteria, it should change the sort component